avgsamplerate.go

```diff
@@ -205,3 +212,14 @@ func (a *AvgSampleRate) LoadState(state []byte) error {
 
 	return nil
 }
+
+func (a *AvgSampleRate) GetMetrics(prefix string) map[string]int64 {
+	a.lock.Lock()
+	defer a.lock.Unlock()
+	mets := map[string]int64{
+		prefix + "_request_count": a.requestCount,
```

Comment:
Should we expect the prefix to include the `_`?  If no prefix is provided it would end up with `_request_count`.

Reply:
Good catch.